apps/dashboard/src/lib/utils/translations/da.json

```
@@ -2502,6 +2505,8 @@
       "email_message_label": "E-mail besked",
       "email_message_placeholder": "f.eks. Vi er stolte af dine fremskridt - fortsæt med at lære!",
       "final_exercise_multiple_attempts_note": "Elever får lov til ubegrænsede forsøg på den valgte øvelse.",
+      "deadline_required": "Certifikatfrist er påkrævet, når certifikater kan downloades.",
```

Biome (2.5.7)

[error] The key deadline_required was already declared.

(lint/suspicious/noDuplicateObjectKeys)